index.html

```diff
@@ -1,16 +1,215 @@
 <!DOCTYPE html>
```

Comment:
## Improve the page layout 

- [ ] Since we are using `bootstrap`, we need to make sure nothing is messed up. The page layout only spans some position of the page and all its contest are confined and centred in the middle of the view width. Would you mind effecting these changes to make our page resemble the original page with the contents of this milestone as an inspiration? Below I have tried to share the comparisons. Have a look 👇 

|[Source Page](https://web.archive.org/web/20210120125445/https://www.newsweek.com/)|[Clone Page](https://tanzila-abedin.github.io/newsweek/)|
|:--:|:--:|
|![newsweek](https://user-images.githubusercontent.com/22345961/109549257-cc121e80-7ade-11eb-85c4-32614b4a9fa6.png)|![newsweek-clone](https://user-images.githubusercontent.com/22345961/109549293-d6341d00-7ade-11eb-9d68-fa50bb705b75.png)|